nebula/ui/components/MZRadioDelegate.qml

```diff
@@ -114,8 +115,15 @@ RadioDelegate {
     MZRadioButtonLabel {
```

Comment:
@peterbenvenuto Selected radio labels in this view and in the server list are currently blue. Was the change here intentional to align with the treatment in DNSettings? Makes sense if so but just wanting to verify. 
Figma on the left, local build on the right.
<img width="892" alt="Screen Shot 2023-03-22 at 2 22 41 PM" src="https://user-images.githubusercontent.com/22355127/227014564-334427b5-dc5c-4f90-ae32-781beda18cf6.png">